sdks/go/pkg/beam/runners/prism/internal/engine/strategy.go

```diff
@@ -573,4 +573,69 @@ func (t *TriggerDefault) String() string {
 	return "Default"
 }
 
-// TODO https://github.com/apache/beam/issues/31438 Handle TriggerAfterProcessingTime
+// TimestampTransform is the engine's representation of a processing time transform.
+type TimestampTransform struct {
```

Comment:
TBH this is possibly "Timestamp Control" from the capability matrix, as it can also affect event time.